buildSrc/build.gradle.kts

```diff
@@ -7,6 +10,18 @@ repositories {
    gradlePluginPortal()
 }
 
+kotlin {
+   jvmToolchain(17)
+}
+
+tasks.withType<KotlinCompile>().configureEach {
+   compilerOptions.jvmTarget.set(JvmTarget.JVM_1_8)
```

Comment:
Btw if the reason to use JDK8 bytecode is android, then you should know Android supports JDK11 libraries for quite some time now. So unless there are other reasons to generate Java 8 bytecode, this can probably be bumped to 11 too

Reply:
We can bump this to 11 then.